src/bundle/Resources/public/scss/core/_custom.tooltip.scss

```diff
@@ -0,0 +1,60 @@
+.ez-tooltip {
+    &.show {
+        opacity: 1
+    }
+
+    &.bs-tooltip-top &__arrow {
```

Comment:
DRY, Personally I would do it as
```
&__arrow {
        height: calculateRem(8px);
}

&.bs-tooltip-top &__arrow {
        &::before {
            border-top-color: $ez-white; 
        }
    }

    &.bs-tooltip-right &__arrow {
        &::before {
            border-right-color: $ez-white; 
        }
    }

    &.bs-tooltip-bottom &__arrow {
        &::before {
            border-bottom-color: $ez-white;
        } 
    }

    &.bs-tooltip-left &__arrow {
        &::before {
            border-left-color: $ez-white;
        } 
    }
```
in order to avoid repeating width declaration four times.